spec/models/system_note/document_help_spec.rb

```diff
@@ -1,3 +1,26 @@
+# == Schema Information
```

Comment:
I'm surprised this comment got autogenerated with this commit here when it wasn't autogenerated with the previous commit! But that's OK.